src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -64,16 +64,64 @@ public Task<IReadOnlyList<ITemplateMatchInfo>> GetTemplatesAsync(IEnumerable<Fun
             return _engineEnvironmentSettings.SettingsLoader.GetTemplatesAsync(criteria, filters ?? Array.Empty<Func<ITemplateInfo, MatchInfo?>>(), cancellationToken);
         }
 
-        public async Task<ICreationResult> CreateAsync(ITemplateInfo info, string name, string outputPath, IReadOnlyDictionary<string, string> parameters, bool skipUpdateCheck, string baselineName)
+        /// <summary>
+        /// Instantiates the template.
+        /// </summary>
+        /// <param name="info">The template to instantiate.</param>
+        /// <param name="name">The name to use.</param>
+        /// <param name="outputPath">The output directory for template instantiation.</param>
+        /// <param name="parameters">The template parameters.</param>
+        /// <param name="baselineName">The baseline configuration to use.</param>
+        /// <returns><see cref="TemplateCreationResult"/> containing information on created template or error occurred.</returns>
+#pragma warning disable RS0027 // Public API with optional parameter(s) should have the most parameters amongst its public overloads
```

Comment:
This is a solid warning.  We're sure about disabling it?

Reply:
yes, the method it complains about is Obsolete and will be removed in .NET 7.